app/controllers/episodes_controller.rb

```diff
@@ -1,11 +1,48 @@
 class EpisodesController < ApplicationController
   load_and_authorize_resource
+  before_action :set_episode, only: %i[edit show]
 
   def index
     @episodes = Episode.order(created_at: :desc).page params[:page]
   end
 
-  def show
+  def show; end
+
+  def edit; end
+
+  def create
+    @episode = Episode.new(episode_params.merge(created_by: current_user))
```

Comment:
Better to set `created_by` like this:
```ruby
@episode = Episode.new(episode_params)
@episode.created_by = current_user
```
This is more clear way for understanding process if it goes bigger